.golangci.yml

```diff
@@ -10,6 +10,8 @@ issues:
 linters:
   enable:
     - wrapcheck
+    - stylecheck
+    - staticcheck
```

Comment:
staticcheck is enabled by default.
https://golangci-lint.run/usage/linters/#enabled-by-default-linters

Reply:
Removed.